sycl/source/handler.cpp

```diff
@@ -202,18 +202,26 @@ event handler::finalize() {
         std::move(MRequirements), std::move(MEvents), MCGType, MCodeLoc));
     break;
   case detail::CG::NONE:
-    throw runtime_error("Command group submitted without a kernel or a "
-                        "explicit memory operation.",
-                        PI_INVALID_OPERATION);
+    if (detail::pi::trace(detail::pi::TraceLevel::PI_TRACE_ALL)) {
+      std::cout << "The empty command group is supported by sycl2020"
+                << std::endl;
+    }
+    break;
```

Comment:
Can we create and return an empty event right here instead of modifying code below?

Reply:
The empty event was moved into switch statement